packages/rancher-istio-1.9/generated-changes/dependencies/kiali/dependency.yaml

```diff
@@ -1,2 +1,2 @@
 workingDir: ""
-url: packages/rancher-kiali-server
+url: https://github.com/rancher/charts/raw/release-v2.5/assets/rancher-kiali-server/rancher-kiali-server-crd-1.32.100.tgz
```

Comment:
```yaml
url: https://github.com/rancher/charts.git
subdirectory: charts/rancher-kiali-server/rancher-kiali-server/1.32.100
commit: 0ae672a2bb1d2679145b2c2297ec6830a5b4365e
```

Reply:
commit is 0ae672a2bb1d2679145b2c2297ec6830a5b4365e

Reply:
Yep that was my bad - fixed in the most recent commits!